at.nis.one_click_backup.yml

```diff
@@ -0,0 +1,30 @@
+{
+    "app-id": "at.nis.one_click_backup",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.6",
+    "sdk": "org.kde.Sdk",
+    "command": "NIS_One-Click-Backup_Qt",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=wayland",
+        "--filesystem=host",
+        "--device=dri"
+    ],
+    "modules": [
+        {
+            name: NIS_One-Click-Backup_Qt,
+            buildsystem: cmake-ninja,
+            #build-commands:
+             #   - ./build-flatpak.sh
+              #  - install -D ../../src/build-flatpak/NIS_One-Click-Backup_Qt /app/bin/NIS_One-Click-Backup_Qt
+            sources: [
+                {
+                    type: dir,
+                    path: ../../src/
```

Comment:
how will it find the source?

Reply:
Oh shit, I forgot to change that. I'm sorry, this is my first Flatpak so I'm basically "learning by doing"

Reply:
Changed manifest to download the sources from GitLab with commit [32eb2a6278c235c5ad180ad40d7d981d2a74c666](https://github.com/flathub/flathub/pull/4864/commits/32eb2a6278c235c5ad180ad40d7d981d2a74c666)